packages/test-sync/src/commonTest/kotlin/io/realm/kotlin/test/mongodb/common/SyncConfigTests.kt

```diff
@@ -29,6 +29,7 @@ import io.realm.kotlin.entities.sync.flx.FlexChildObject
 import io.realm.kotlin.entities.sync.flx.FlexEmbeddedObject
 import io.realm.kotlin.entities.sync.flx.FlexParentObject
 import io.realm.kotlin.ext.query
+import io.realm.kotlin.internal.platform.PATH_SEPARATOR
```

Comment:
Hmm. Not for this PR, but unfortunate #naming ... `File.pathSeparator` is usually for separating multiple path entries and not the directory separator 🙃